src/com/javabasic/lesson6/homework/Task1.java

```diff
@@ -0,0 +1,24 @@
+package com.javabasic.lesson6.homework;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int min = 1;
+        int max = 100;
+
+        printMinNumber(random(min, max), random(min, max), random(min, max));
+    }
+
+    public static int random (int min, int max) {
+        return (int) (Math.random() * (max - min) + min);
+    }
+
+    public static void printMinNumber (int number1, int number2, int number3) {
```

Comment:
<img width="1129" alt="image" src="https://github.com/andry80/Java_Basic/assets/2835251/9c327436-b6d0-494a-832e-2952ac826e32">
What if you have equal numbers?